db/migrate/20240722202054_create_oauth_token_exchange_grants.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+##
+#  Creates the oauth_token_exchange_grants table
+class CreateOAuthTokenExchangeGrants < ActiveRecord::Migration[7.0]
+  def change
+    table.references :user, null: false, foreign_key: true
```

Comment:
Hmm, the `create_table` block somehow got removed. You'll need to wrap lines 7-10 in this block:

```ruby
create_table :oauth_token_exchange_grants, id: :uuid do |table|
  ...
end
```

Reply:
Oops my bad! Added that back in